zk/zk_test.go

```
@@ -93,6 +97,153 @@ func TestCreate(t *testing.T) {
+	ts, err := StartTestCluster(t, 3, nil, logWriter{t: t, p: "[ZKERR] "})
+	requireNoError(t, err, "failed to setup test cluster")
+	defer ts.Stop()
+
+	// start and add a new server.
+	tmpPath, err := ioutil.TempDir("", "gozk")
+	requireNoError(t, err, "failed to create tmp dir for test server setup")
+	defer os.RemoveAll(tmpPath)
+
+	startPort := int(rand.Int31n(6000) + 10000)
+
+	srvPath := filepath.Join(tmpPath, fmt.Sprintf("srv4"))
```

golangci-lint (2.12.2)

[error] S1039: unnecessary use of fmt.Sprintf

(staticcheck)